packages/DataHeader/stories/DataHeader.screener.stories.js

```diff
@@ -0,0 +1,180 @@
+            <DataHeader
+              backgroundColor="magenta"
+              color="white"
+              icons={DataHeader.icons}
+              label="name"
+              type={DataHeader.types.type.TEXT}
+              renderActions={() => <Menu type="name" />}
+            />
+          )}
+          cell="name"
+        />
+        <Table.ColumnDefinition
+          header={() => (
+            <DataHeader
+              backgroundColor="cyan"
+              icons={DataHeader.icons}
+              label="Income"
+              type={DataHeader.types.type.NUMERIC}
+            />
+          )}
+          cell={({ row }) => row.income}
+        />
+        <Table.ColumnDefinition
+          header={() => (
+            <DataHeader
+              backgroundColor="black"
+              color="white"
+              icons={DataHeader.icons}
+              label="Value"
+              type={DataHeader.types.type.DATE}
+            />
+          )}
+          cell={({ row }) => row.taxes}
+        />
+        <Table.ColumnDefinition
+          header={() => (
+            <DataHeader
+              backgroundColor="orange"
+              color="black"
+              icons={DataHeader.icons}
+              label="Value"
+              type={DataHeader.types.type.TIME}
+            />
+          )}
+          cell={({ row }) => row.taxes}
+        />
+        <Table.ColumnDefinition
+          header={() => (
+            <DataHeader
+              backgroundColor="white"
+              color="red"
+              icons={DataHeader.icons}
+              label="Value"
+              type={DataHeader.types.type.DATE_TIME}
+            />
+          )}
+          cell={({ row }) => row.taxes}
+        />
+      </Table>
+      <hr />
+      <Table data={data}>
+        <Table.ColumnDefinition
+          width="150px"
+          header={() => <DataHeader backgroundColor="magenta" color="white" label="name" />}
+          cell="name"
+        />
+        <Table.ColumnDefinition
+          header={() => <DataHeader backgroundColor="cyan" label="Income" />}
+          cell={({ row }) => row.income}
+        />
+        <Table.ColumnDefinition
+          header={() => <DataHeader backgroundColor="black" color="white" label="Value" />}
+          cell={({ row }) => row.taxes}
+        />
+        <Table.ColumnDefinition
+          header={() => <DataHeader backgroundColor="orange" color="black" label="Value" />}
+          cell={({ row }) => row.taxes}
+        />
+        <Table.ColumnDefinition
+          header={() => <DataHeader backgroundColor="white" color="red" label="Value" />}
+          cell={({ row }) => row.taxes}
+        />
+      </Table>
+    </Story>
+  );
+};
+
+const icons = {
+  ...DataHeader.icons,
+  [DataHeader.types.type.DATE_TIME]: <CaretUp />,
+};
+
+export const TypesScreener = () => {
+  return (
+    <Story>
+      <h3>TEXT</h3>
+      <DataHeader icons={DataHeader.icons} label="name" type={DataHeader.type.types.TEXT} />
```

Comment:
```suggestion
      <DataHeader icons={DataHeader.icons} label="name" type={DataHeader.types.type.TEXT} />
```

Reply:
Not sure if that's the right fix, but the story is currently broken

Reply:
yes its, I already fixed it thank you :)